instrumentation/http.js

```diff
@@ -113,6 +113,29 @@ function instrumentHTTP(mod) {
               'response.status_code': res.statusCode,
               ...responseHeaderFields(res.headers),
             });
+
+            let responseBody = '';
+            res.on('data', chunk => {
+              responseBody += chunk;
+            });
+
+            res.on('end', () => {
+              if (res.statusCode && res.statusCode >= 400) {
+                if (responseBody.length > 1000) {
+                  responseBody = responseBody.slice(0, 1000);
+                }
+                span.addContext({ 'response.body': responseBody });
+              }
+
+              beeline.finishSpan(span, 'request');
+            });
+          });
+
+          req.once('error', err => {
+            span.addContext({
+              'error.message': err.message,
```

Comment:
I would probably just use `error` as the key for the message, so that it gets picked up by `derived.error`
```suggestion
              'error': err.message,
```